src/datetime/getTimezone.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Returns the current timezone
+ * @return timezone
+ */
+
+import { timezones } from "./timezone";
+import { ConfigurationState } from "../config/configurationState";
+import { storageKey, fromString } from "../datetime/timezone";
```

Comment:
This `import` resolves to `"./timezone"`, as the `import` in line 6. ¿Merge into one single line?